src/sparseml/transformers/question_answering.py

```diff
@@ -771,8 +752,8 @@ def compute_metrics(p: EvalPrediction):
     trainer = QuestionAnsweringTrainer(
         model=model,
         model_state_path=model_args.model_name_or_path,
-        recipe=data_args.recipe,
-        recipe_args=data_args.recipe_args,
+        recipe=training_args.recipe,
```

Comment:
Looks like this should be mimicked in the `masked_language_modeling.py` and `token_classification.py`

Reply:
Fixed